docs/DeveloperGuide.md

```diff
@@ -324,13 +329,149 @@ Below are instructions to perform manual testing of the application. Please refe
-2. Test Case: `del mod CS2113T` <br>
+1. Prerequisite: There are existing modules in the application.
+2. Assumption: You have a module `CS2113T` added, but not `CS2101`.
+3. Note: If you have added tasks to a module, deleting that module will prompt a confirmation to delete. Typing `yes` will delete that module. 
+4. Test Case: `del mod CS2113T` <br>
    Expected: The module CS2113T is deleted.
-3. Test Case: `del mod CS2101` <br>
-   Expected: If CS2101 does not exist, no module is deleted. Error details in the message shows that there are no such module. 
+5. Test Case: `del mod CS2101` <br>
+   Expected: No module is deleted. Error details in the message shows that there are no such module. 
+
+<br>
+
+### Deleting a task
+1. Prerequisite: There are existing tasks in the application.
+2. Assumption: You have the module `CS2113T` added with at least one task added. 
+3. Test Case: `del task 1 -m CS2113T` <br>
+   Expected: If there exists a module CS2113T, and it has at least one task, the first task will be deleted.
+4. Test Case: `del task -1` <br>
+   Expected: No task is deleted. Error details in the message shows that the task number is invalid.
+
+<br>
+
+### Editing a module
+1. Prerequisite: There are existing modules in the application. 
+2. Assumption: You have the module `CS2113T` added.
+3. Test Case: `edit mod CS2113T -d "Changed"` <br>
+   Expected: The description of CS2113T is set to `Changed`.
+4. Test Case: `edit mod CS2113T -t "2 hours"` <br>
+   Expected: The module remains unchanged. Error details in the message shows that the module description is missing.
+
+<br>
+
+### Editing a task
+1. Prerequisite: There are existing tasks in the application.
+2. Assumption: You have at least one task in your `General Tasks`.
+3. Test Case: `edit task 1 -d "Changed"` <br>
+   Expected: The description of the first task in `General Tasks` is set to `Changed`.
+4. Test Case: `edit task 1 -d "Changed" -t "2 hours"` <br>
+   Expected: The task remains unchanged. Error details in the message shows that there is an excess argument of `-t "2 hours"`.
+
+<br>
+
+### Setting grade for a module
+1. Prerequisite: There are existing modules in the application.
+2. Assumption: You have a module `CS2113T`.
+3. Test Case: `grade CS2113T A+` <br>
+   Expected: The grade of `CS2113T` has been set to `A+`.
+4. Test Case: `grade CS2113T E` <br>
+   Expected: No grade is set. Error details in the message shows that `E` is an invalid module grade.
+
+<br>
+
+### Calculating GPA
+1. Prerequisite: There are existing modules in the application.
+2. Assumption: You have a module `CS2113T` of `4` modular credits and grade `A+` and a module `CS2101` of `4` modular credits and grade `B`.
+3. Test Case: `gpa` <br>
+   Expected: Your `gpa` is calculated as `4.25`.
+
+<br>
+
+### Showing Help
+1. Test Case: `help` <br>
+   Expected: A message for format of the `help` command is shown.
+2. Test Case: `help add"` <br>
```

Comment:
there shouldnt be a `"`. It should just be `help add`.

Reply:
removed the error